### DISCUSSION TITLE
X Touch Compact issues

### DISCUSSION THREAD

**Discussion by yomozingo** [rsjaffe/MIDI2LR#667] [Problems]
I'm am having a very difficult time getting my Behringer X Touch Compact to correctly sync with Midi2LR.  Maybe I'm missing something?

For example, when I move the first slider it pops up 2 Midi commands... CC:28 and CC:111.  I have been trying to assign "Exposure" but I'm unable to get it to work properly.  I have setting at 0-127.  Why does is assign 2 midi commands for each slider?

I have read this article several times.  https://github.com/rsjaffe/MIDI2LR/wiki/MIDI-Controller-Setup

I just have no idea what is wrong.  

Can anyone help me trouble shoot this please?

Thanks!

**Comment by antzpantz** [comment]
Maybe you need to configure the controller using the Behringer software?

**Reply by yomozingo** [reply]
Thank you for the response.  What is the Behringer software?

**Reply by yomozingo** [reply]
Just wanted to follow-up on this message since I never gat a response.  Thank you!

**Comment by rsjaffe** [comment]
Is the slider set up to send NRPN codes? That’s one reason it could do that.

**Reply by yomozingo** [reply]
How do I determine whether or not it is set up to send NRPN codes?  Thanks.

**Reply by yomozingo** [reply]
Just wanted to follow-up on this message since I never gat a response.  Thank you!

**Comment by antzpantz** [comment]
Sorry, I forgot to reply. The X-Touch Editor software is used to configure
your controller. That should also determine if your controller is set up to
send NRPN codes.

https://www.behringer.com/product.html?modelCode=P0B3L

**Reply by yomozingo** [reply]
Thank you!

**Comment by Papp-Anti** [comment]
Hello! I don't know English, so google translates! I just bought a compact too. I'm stuck on setting it up. The rsjaffe/midi2lr Wiki says so "Go to Files→Plugin Extras→MIDI2LR Keywords. MIDI2LR starts indexing your keywords as soon as Lightroom is started. If you have thousands of keywords, it may take a minute or two. If you open this menu before it is all indexed, you'll get a message telling you how many keywords have been indexed so far, and you'll then receive a bezel notification when the keyword indexing is completed. " On the other hand, nothing loads for me, no matter how long I wait. The keyword fields are empty. Actually, I don't know if I can set it here so that I can assign watermarks or user export profiles to buttons on the midi controller. does anyone have any ideas? Thank you for helping me!
